script.js

```diff
@@ -0,0 +1,70 @@
+console.log('js file linked!');
+const resetBut = document.getElementById('resetBut');
+const squares = Array.from(document.getElementsByClassName('square'));
+const playerText = document.getElementById('playerText');
+const spaces= [null, null, null, null, null, null, null, null, null];
+
+console.log(squares);
+
+const oTurn = "O";
+const xTurn = "X";
+
+let currentTurn;
+
+
+const clickCon = (e) => {
+    const id = e.target.id;
+    console.log(id);
+
+    if(!spaces[id]) {
+        spaces[id] = currentTurn;
+        e.target.innerText = currentTurn;
+    
+        currentTurn = currentTurn === oTurn ? xTurn : oTurn;
+    }
+}
+
+squares.forEach((square, index) => {
+    square.addEventListener('click', clickCon);
+    })
+
+    const theWinner = () => {
```

Comment:
This indentation gave me a jump scare because I thought you were redefining this function 9 times inside your forEach! (and in a throwaway scope too!)